app/controllers/schools/find_out_more_controller.rb

```diff
@@ -10,8 +10,14 @@ def show
       @activity_types = activity_type_filter.activity_types.limit(3)
       @alert = @find_out_more.alert
       @content = TemplateInterpolation.new(@find_out_more.content_version).interpolate(:page_title, :page_content, with: @alert.template_variables)
-      @charts = @alert.charts
+      @chart = chart(@find_out_more.content_version.chart_type)
       @tables = @alert.tables
     end
+
+  private
+
+    def chart(chart_type)
+      chart_type == 'none' || chart_type.nil? ? nil : chart_type
```

Comment:
As per https://github.com/BathHacked/energy-sparks/pull/573/files#r278846110 we should be looking at using the variable name stored in the content to cross-reference against the alert's chart_data to get the chart name, like we do with template variables.